examples/http-client-header-parameter/http_client_header_parameter.md

```diff
@@ -1,6 +1,6 @@
 # HTTP client - Header parameter
 
-The http module provides support for sending outbound request headers as resource method arguments. The headers can be provided as `map<string|string[]>` to the resource method. 
+The HTTP client supports sending outbound request headers along with the request payload. These headers can be passed as an argument in the client resource method call. The headers should be provided as a `map` where the keys represent the header names and the entries represent the header values. The header values can be `string` or `string[]`. Use this when you want to send additional headers as part of the request.
```

Comment:
```suggestion
The `http:Client` supports sending outbound request headers along with the request payload. These headers can be passed as an argument in the client resource method call. The headers should be provided as a `map` where the keys represent the header names and the entries represent the header values. The header values can be `string` or `string[]`. Use this when you want to send additional headers as part of the request.
```

Reply:
In other BBEs we have used like `The HTTP client`. So which should be followed?
@shafreenAnfar @chamil321

Reply:
+1 to quote it